tests/app/idp/requirements.txt

```
@@ -1,5 +1,5 @@
-Django>=3.2,<4.2
+Django>=3.2,<=4.2
```

```suggestion
Django>=4.2,<=5.1
```

Shouldn't the versions match those [here](Django 4.2, 5.0 or 5.1)?